doc/getting-started/how-helix-works.md

```diff
@@ -0,0 +1,19 @@
+# How does Helix work?
+
+This page takes a behind-the-scenes look at the overall Helix architecture and describes its main components.
+
+![Helix Architecture Overview](/assets/helix-overview.png)
+
+Helix uses the Fastly CDN to deliver content and deploys Helix-specific _VCL edge code_ to process client requests.
```

Comment:
I would suggest labeling these sections based on the "column" of the diagram. So for example, the heading over this sentence would be "Fastly CDN" - so that I can quickly map the section of the document to the area of the diagram.

Reply:
I see your point but I'll take a slightly different approached, setting the terms in **bold** when they map to a diagram element, like **Fastly CDN**. I'll comment here when those changes are pushed.

Sections wouldn't really work as I/O Runtime for example is mentioned in several paragraphs. And also I want the reader to think in terms of the logical flow of the request as opposed to where the components run.